### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by henryre**
**Test plan**
* Testing `pip` on a base Ubuntu AWS machine
* Testing `conda` on a AWS Deep Learning AMI machine

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -133,3 +133,6 @@ dask-worker-space/
 
 # Generic data directories
 **/data/*
+
+# Modeling
+checkpoint/
```
Comment (vincentschen) [inline .gitignore]: the default is `checkpoints` https://github.com/HazyResearch/snorkel/blob/redux/snorkel/classification/training/loggers/checkpointer.py#L38

File: README.md [hunk 2]
```diff
@@ -17,8 +17,27 @@ All other tutorials assume that you have already completed that tutorial and are
...
+As with Snorkel, our tutorials require Python 3.6+.
+If you're looking to quickly get started with a tutorial, we recommend using
+our [Docker setup](#docker).
+If you want to install things yourself using `pip` or `conda`, you can follow
+our [installation steps](#install) below instead.
+
+*A quick note for Windows users*
+
+If you're using Windows, we highly recommend using the [Docker setup](#docker)
+or the [Linux subsystem](https://docs.microsoft.com/en-us/windows/wsl/faq).
```
Comment (vincentschen) [inline README.md]: I think A. tried the Linux subsystem, and it didn't work so hot—but certainly better than whatever other defaults

File: README.md [hunk 3]
```diff
@@ -52,20 +76,61 @@ Running a tutorial has three required steps if you're installing yourself:
...
+```
+
+</p>
+</details>
+
+Then in the browser tab that opens, navigate to a `.ipynb` file you would like
+to run &mdash; such as `spam/spam_tutorial.ipynb` &mdash; and execute the
+cells in sequence.
+
+Alternatively, you can run the tutorial as a script by calling `python3` on the corresponding `.py` file directly (e.g. `python3 spam/01_spam_tutorial.py`).
```
Comment (vincentschen) [inline README.md]: maybe a quick plug for `jupytext` here? folks may not be familiar

File: .gitignore (hunk 1 above)
Comment (henryre) [reply .gitignore]: I think this is Keras maybe? Idk evidently something I did made this dir

Comment (vincentschen) [reply .gitignore]: possibly—which tutorial were you running? 
in any case, let's either include both checkpoint dirs or just one of them and make it consistent across tutorials (cc @bhancock8, for visibility when we check consistency)

File: README.md (hunk 2 above)
Comment (henryre) [reply README.md]: had to do with particularities of conda install...can't defend against everything (but Docker can)

**Pull request merged**